modules/service/src/main/scala/lucuma/odb/graphql/input/SpectroscopyScienceRequirementsInput.scala

```diff
@@ -9,34 +9,31 @@ import eu.timepit.refined.types.numeric.PosInt
 import lucuma.core.enums.FocalPlane
 import lucuma.core.enums.SpectroscopyCapabilities
 import lucuma.core.math.Angle
-import lucuma.core.math.SignalToNoise
 import lucuma.core.math.Wavelength
+import lucuma.core.model.ExposureTimeMode
 import lucuma.odb.data.Nullable
 import lucuma.odb.graphql.binding.Matcher
 import lucuma.odb.graphql.binding.ObjectFieldsBinding
 import lucuma.odb.graphql.binding.PosIntBinding
-import lucuma.odb.graphql.binding.SignalToNoiseBinding
 import lucuma.odb.graphql.binding.enumeratedBinding
 
 final case class SpectroscopyScienceRequirementsInput(
   wavelength:         Nullable[Wavelength],
   resolution:         Nullable[PosInt],
-  signalToNoise:      Nullable[SignalToNoise],
-  signalToNoiseAt:    Nullable[Wavelength],
+  exposureTimeMode:   Nullable[ExposureTimeMode],
   wavelengthCoverage: Nullable[Wavelength],
   focalPlane:         Nullable[FocalPlane],
   focalPlaneAngle:    Nullable[Angle],
   capability:         Nullable[SpectroscopyCapabilities]
 )
 
-object SpectroscopyScienceRequirementsInput {
+object SpectroscopyScienceRequirementsInput:
 
   val Default: SpectroscopyScienceRequirementsInput =
     SpectroscopyScienceRequirementsInput(
       wavelength         = Nullable.Null,
```

Comment:
Did we discuss that perhaps here the defaults should be `Absent`?

Reply:
Yeah, we might want to circle back and change that.